data/schema.graphql

```diff
@@ -9356,10 +9356,11 @@ type ViewingRoom {
 
   # Body copy
   body: String
+  distanceToClose(short: Boolean! = false): String
```

Comment:
The `short` option will be used for iOs?

Reply:
Yes, and in general there are small differences. Like `short` shows `soon` for 40 days, https://github.com/artsy/metaphysics/pull/2486/files#diff-e9a6ef1ded9b9a8e03c9320518a09a23R151, and the long version shows `null`, https://github.com/artsy/metaphysics/pull/2486/files#diff-e9a6ef1ded9b9a8e03c9320518a09a23R118. Basically the `short` version is the basic info about the distance we would get at a glance, like in the tag, and the long version is the full info about the distance. So the short might say `soon`, and the long might say `29 days`.